### DISCUSSION TITLE
Redudant code in AutoOrientProcessor?!

### DISCUSSION THREAD

**Discussion by Mertsch** [SixLabors/ImageSharp#2750] [Ideas] [answered]
This code
https://github.com/SixLabors/ImageSharp/blob/ede2f2d2d1e567dea74a44a482099302af9ed14d/src/ImageSharp/Processing/Processors/Transforms/Linear/AutoOrientProcessor%7BTPixel%7D.cs#L91C79-L91C96

Since `IExifValue<ushort>` is typed now (compared to some years ago)

Is this really necessary?
https://github.com/SixLabors/ImageSharp/blob/ede2f2d2d1e567dea74a44a482099302af9ed14d/src/ImageSharp/Processing/Processors/Transforms/Linear/AutoOrientProcessor%7BTPixel%7D.cs#L103

It was from a time past https://github.com/SixLabors/ImageSharp/issues/271

**Comment by JimBobSquarePants** [comment, marked as answer]
Looks like it's redundant yes. Well spotted!
